os/BUILD

```diff
@@ -111,3 +111,14 @@ constraint_value(
     name = "chromiumos",
     constraint_setting = ":os",
 )
+
+constraint_value(
```

Comment:
This will lead to a mess. trusty is not an OS. It is a linux release. 
Can we keep this internal for now and not pollute the global namespace?

Reply:
cc @deeglaze
https://source.android.com/docs/security/features/trusty says trusty's got its own kernel, not linux.
It's also got its own OSS repository, so I can move the constraint there.
The only problem I see with this is that previously you mentioned as criteria that if OS is public it should go into this repository. Now I'm confused what the criteria is? Number of users? Or arbitrary?

Reply:
Remove it.

Reply:
My apologies.   I see now that trusty is an android specific mini-os.  https://source.android.com/docs/security/features/trusty
It is unfortunate that the name clashes with the Linux trusty release.

If we were going to add this here, lets call it `android_trusty`.
But, i would prefer that it was defined with rules_android, because it is only applicable for teams using android.